api/src/models/team.model.js

```diff
@@ -21,6 +21,9 @@
     },
      twitterHandle: {
         type: String
+    },
+     githubToken: {
+        type: String
```

Comment:
Are you adding the post and get request for this in another PR?

Reply:
This patch request was created for the functionality of posting the GitHub token to a current team. You can just pass in a JSON containing only the githubToken and send a patch request to store the token (refer to images above). There's already a GET request for team/id endpoint which will return a json of all the details for that team so you can use this to get the token.

Reply:
Ahh right, sorry didn't see the screenshot hahaha